### PR TITLE
Add a contact form

### PR DESCRIPTION
**Pull request by NJ-2020**
Dear reviewer,

This is my final mobile version portfolio.
I have fixed the linter errors and update HTML and CSS files.

Please let me know your feedbacks

Thanks
Novell

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,27 +113,76 @@ <h3>LET'S CONNECT</h3>
...
         <div class="nnjjn a">
-          <h3 class="text-languages">Skills</h3><img src="IMG/Union.svg" alt="arrright" class="arrright">
+          <h3 class="text-languages">Skills</h3>
+          <img src="IMG/Union.svg" alt="arrright" class="arrright" />
         </div>
       </section>
     </div>
+    <div class="container_contact">
+      <div class="content">
+        <h1 class="h1_contact">Contact me!</h1>
```
Comment (Davidosky007) [inline index.html]: `h1` tags are used for main tiltes (first heading of a web page) and this is not a main title so `h2`,`h3`, etc can be used visit this [Link](https://www.w3schools.com/tags/tag_hn.asp) for more information on HTML headings

File: index.html [hunk 2]
```diff
@@ -114,27 +113,76 @@ <h3>LET'S CONNECT</h3>
...
+          class="forms-input"
+          action="https://formspree.io/f/mayajwnb"
+          method="post"
+        >
+          <input
+            type="text"
+            class="input"
+            maxlength="30"
+            required
+            placeholder="Full Name"
```
Comment (Davidosky007) [inline index.html]: I suggest you add a name attribute to this input as this is best practice

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @yahyaelganyni1 @NJ-2020, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest your Pull Request have a descriptive title telling you what changes are inside.
* I suggest your  Pull Request have a short summary describing introduced changes in more detail.
* I think your commit messages can begin with capital letters it is best practice read [This](https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines) for further information.
* Please check the below and make the necessary changes, submit for review with this same PR 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by NJ-2020** [comment]
Dear reviewer,

I add a contact form to complete this portfolio - a mobile version.
The changes I made include:
- Create forms using HTML5 validations.
- I also change the h1 tag to h2 to apply your feedback
- I add a name attribute to the input to apply best practice as you suggest

Regards
Novell

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi @NJ-2020 & @yahyaelganyni1 👋 👋 , 

## STATUS: APPROVED ✔️ ✔️ 

Your project is complete! 👏 👏 All changes requested by the previous reviewer have been implemented. There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

[OPTIONAL]
- Your README screenshot is broken please fix that in order to make your repo look professional.
- The comment you added last I believe was supposed to be added at the pull request top comment. Just click on edit and replace this message with the existing one.

#### ._Cheers and Happy coding!👏👏👏._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**